src/ConfigurationProcessor.php

```diff
@@ -26,7 +26,12 @@ final class ConfigurationProcessor
     /**
      * @var int the time to wait before check the status of a process (in ms)
      */
-    const DEFAULT_PROCESS_UPDATE_FREQUENCY = 200;
+    const DEFAULT_PROCESS_UPDATE_FREQUENCY = 2000;
```

Comment:
spamming the output every 200ms for an operation of minutes is not really interesting 👍